microshift_release_notes/microshift-4-16-release-notes.adoc

```diff
@@ -63,8 +63,13 @@ See the following list for details:
 
 [id="microshift-4-16-custom-cert-auths"]
 ==== Customizable certificate authorities for the API server are supported
-With this release, you can configure a custom server certificate that has been issued by an external certificate authority (CA). The default API server certificate is issued by an internal {microshift-short} cluster CA. You can now replace this certificate with one that is issued by a CA that clients trust.
-//TODO add xref after section merges
+With this release, you can configure a custom server certificate that has been issued by an external certificate authority (CA). The default API server certificate is issued by an internal {microshift-short} cluster CA. You can now replace this certificate with one that is issued by a CA that clients trust. See xref:../microshift_configuring/microshift-custom-ca.adoc#microshift-custom-ca[Configuring custom certificate authorities] for details.
+
+[id="microshift-4-16-audit-logging-config"]
+==== Configurable policies for log file rotation and retention
+
+You can now configure audit logging policies to manage the retention policies for log files, ensuring that edge devices with limited storage capacities are not hampered by logging data accumulation. To configure audit log policies, use settings such as a maximum file size limit and maximum retained files to set a limit on log storage size. You can also choose an audit policy profile to specify the data collected.
```

Comment:
```suggestion
You can now configure audit logging policies to manage the retention policies for log files, ensuring that edge devices with limited storage capacities are not hampered by the accumulation of logging data. To configure audit log policies, use settings such as a maximum file size limit and maximum retained files to set a limit on log storage size. You can also choose an audit policy profile to specify the data collected.
```

Reply:
I wasn't sure if any of the settings mentioned needed backtics because they are literals. For example, `log storage` size.

Reply:
Nah, the setting names are like `maxFileSize`, etc. I didn't use those here.